system-parachains/people/people-polkadot/src/tests.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright (C) Parity Technologies and the various Polkadot contributors, see Contributions.md
+// for a list of specific contributors.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::xcm_config::LocationToAccountId;
+use cumulus_primitives_core::relay_chain::AccountId;
+use sp_core::crypto::Ss58Codec;
+use xcm::prelude::*;
+use xcm_runtime_apis::conversions::LocationToAccountHelper;
+
+const ALICE: [u8; 32] = [1u8; 32];
+
+#[test]
+fn location_conversion_works() {
+	let alice_32 = AccountId32 { network: None, id: AccountId::from(ALICE).into() };
+	let bob_20 = AccountKey20 { network: None, key: [123u8; 20] };
+
+	// the purpose of hardcoded values is to catch an unintended location conversion logic change.
+	struct TestCase {
+		description: &'static str,
+		location: Location,
+		expected_account_id_str: &'static str,
+	}
+
+	let test_cases = vec![
```

Comment:
Why are the test-cases not defined in a shared crate? They seem to be copy&pasted?

Reply:
There is a subtle difference among these tests -- DescribeAccountId32Terminal is different in many of the cases. 

If I were to refactor the tests I'd rather rewrite those into macros, as Adrian suggested a few comments before. I'll leave it as a follow-up for myself once I fully onboard.

Reply:
Unless you're talking about people folder specifically, though I would not do it for a single folder...